modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -1,6 +1,6 @@
 [[troubleshooting]]
 = Troubleshooting
-:page-aliases: troubleshooting/faqs.adoc, troubleshooting/security.adoc, troubleshooting/optimizing-create-operations.adoc, appendix/preventing-overfetching.adoc, appendix.adoc
+:page-aliases: troubleshooting/faqs.adoc, troubleshooting/security.adoc, troubleshooting/optimizing-create-operations.adoc, appendix.adoc
```

Comment:
Is there a reason why you removed `appendix/preventing-overfetching.adoc`? It could be useful in case someone tries to reach out to that page and then is instead redirected to this one.

Reply:
That section is no longer relevant, but I was not too knowledgeable about how page-aliases work, so maybe we should keep it